seqr/views/utils/orm_to_json_utils.py

```diff
@@ -245,6 +258,9 @@ def _process_result(result, individual):
             result['sampleGuids'] = [s.guid for s in individual.sample_set.all()]
             result['igvSampleGuids'] = [s.guid for s in individual.igvsample_set.all()]
 
+    kwargs = {
+        'additional_model_fields': _get_case_review_fields(individuals[0], individuals[0].family.project, user) or []
```

Comment:
`or []` can be removed if `_get_case_review_fields()` return `[]` when there is no permission.